schedule/yast/btrfs/btrfs_sle_libstorage-ng.yaml

```diff
@@ -35,6 +35,7 @@ schedule:
   # Called on BACKEND: qemu
   - {{grub_test}}
   - installation/first_boot
+  - console/check_resume
```

Comment:
Hmm, I would actually not schedule this module here, as it will be used for all setups. Maybe conditional schedule?

Reply:
Mm, you're right. I'll do this.